app/controllers/items_controller.rb

```diff
@@ -19,6 +16,29 @@ def create
 
   end
 
+  def show
+    render body: 'User not exist', status: 404 unless (@item = Item.where(id: params[:id]).first)
+  end
+
+  def edit
```

Comment:
By the way, do you know about scaffolds generator in rails (try it in separate PR):

```
rails generate scaffold Item name:string price:decimal
```

and check its docs:

```
rails generate scaffold --help
```

After that the next step is to start using [decent_exposure](https://github.com/hashrocket/decent_exposure) gem